library/src/lib/utils/date-time-format-parsers.ts

```diff
@@ -0,0 +1,128 @@
+import { FdDate } from '../calendar/models/fd-date';
```

Comment:
You are hand-parsing date time here. 

We had agreed all libraries fundamental libraries would use momentjs.